src/components/Panel/Panel.less

```diff
@@ -5,6 +5,7 @@
 // --------------------------------------------------
 // Panel styles
 
+@ms-Panel-mdWidth: 341px;
```

Comment:
nit: should probably be 340px. Then just @ms-Panel-mdWidth + 1 down on line 138 for ms-Panel-main width if it has to be 341px.